src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue

```diff
@@ -156,6 +157,10 @@ const handleChangeEditContent = (propertyName: string) => {
     state.selectedVariable = propertyName;
     state.contentType = 'EDIT';
 };
+const handleChangeCloneContent = (propertyName: string) => {
+    state.selectedVariable = propertyName;
+    state.contentType = 'ADD';
```

Comment:
suggestion:
How about `CLONE`?
Then you should check both the `ADD` and `CLONE` cases below.
```typescript
if (state.contentType === 'ADD' || state.contentType === 'CLONE') {
```
But I think this is easier to understand the flow.

Reply:
I think so too. But CLONE is also ADD case... hmm.. 
In CLONE case, Heading still have ADD title and ADD form.
So I need to think little more.

Reply:
I fixed it!